lib/pf/services/manager/winbindd_child.pm

```diff
@@ -74,6 +74,12 @@ sub build_namespaces(){
         my $OUTERLOGDIRECTORY="$CHROOT_PATH/$LOGDIRECTORY";
         my $OUTERRUNDIRECTORY="$CHROOT_PATH/var/run/samba$domain";
         my $PIDDIRECTORY="$var_dir/run/$domain";
+        my $DOMAIN_IF="$domain-b";
+        my $DOMAIN_IFCFG_VIRT_INT="/etc/sysconfig/network-scripts/ifcfg-$DOMAIN_IF";
+        unless (-e $DOMAIN_IFCFG_VIRT_INT) {
+           my $cmd='echo -e "DEVICE='.$DOMAIN_IF.'\nNM_CONTROLLED=no" |sudo tee '.$DOMAIN_IFCFG_VIRT_INT;
```

Comment:
This is more for coherence then anything else. (all packetfence linked interfaces are told to not be managed by nm).